feature/s3/transfermanager/api_op_DownloadObject.go

```diff
@@ -776,14 +790,6 @@ func (d *downloader) incrWritten(n int64) {
 	d.written += n
 }
 
-// getTotalBytes is a thread-safe getter for retrieving the total byte status.
-func (d *downloader) getTotalBytes() int64 {
```

Comment:
[nit] was this just unused all around? I suspect we're removing this since the actual solution is "use a progress listener", right?

Reply:
Yeah, this was unused. I assume it was ported over from v1 but refactoring made it unnecessary.